spec/FSi/Bundle/DoctrineExtensionsBundle/Twig/Extension/AssetsSpec.php

```diff
@@ -13,8 +13,13 @@
 use FSi\DoctrineExtensions\Uploadable\File;
 use PhpSpec\ObjectBehavior;
 use Prophecy\Argument;
-use Symfony\Bundle\TwigBundle\Extension\AssetsExtension;
-use Symfony\Component\Routing\RequestContext;
+
+class AssetsExtension
+{
+    public function getAssetUrl($path)
```

Comment:
This method is implemented in https://github.com/symfony/TwigBridge/blob/master/Extension/AssetExtension.php (>=2.7) or https://github.com/symfony/TwigBundle/blob/2.6/Extension/AssetsExtension.php (<2.7). This class allows compatibility with both versions.